src/Grapevine/Server/Router.cs

```diff
@@ -531,35 +529,44 @@ public void Route(object state)
             var context = state as IHttpContext;
             if (context == null) return;
```

Comment:
Yeah, I completely expect it to be an IHttpContext, however it gets called from [here](https://github.com/sukona/Grapevine/blob/public-folder/src/Grapevine/Server/RestServer.cs#L370) and will be passed as type `object`, so I have to have it that way in the method signature.

While it is unlikely to ever be anything else, it is a public method, and could potentially be called by anyone with anything. If the cast fails, `context` will be null, and it should just return at that point. I see no reason to raise an exception there, but I could be persuaded otherwise.